.github/workflows/build.yml

```diff
@@ -16,7 +16,7 @@ jobs:
     tests:
         runs-on: ubuntu-18.04
 
-        name: "PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, MySQL ${{ matrix.mysql }}"
+        name: "PHP ${{ matrix.php }}, Symfony ${{ matrix.symfony }}, Node ${{ matrix.node }}, MySQL ${{ matrix.mysql }}"
```

Comment:
Please rebase your code, it's already done in current master.

Reply:
@rimas-kudelis could you please rebase PR.

Reply:
@diimpp, I've done it once already. It's your turn now.